microsoft-edge/webview2/concepts/multi-profile-support.md

```diff
@@ -0,0 +1,279 @@
+---
+title: Support multiple profiles under a single user data folder
+description: How to have different profiles of a WebView2 app running with different profiles under a single user data folder to achieve browsing data separation.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: webview
+ms.date: 10/11/2022
+---
+# Support multiple profiles under a single user data folder
+
+To achieve data separation, a WebView2 app can choose to use different user data folders for different WebView2 controls. However, in such way you'll have to be running multiple WebView2 runtime instances (each including a browser process and a bunch of child processes), which means much more consumption for system resources including memory, CPU footprint, disk space, so it is not desirable.
+
+Now we have a more desirable solution: multiple profile support, which enables a WebView2 app to have multiple profiles under a single user data folder, each profile has a dedicated profile folder to save browser data, which means separate browsing data storage, such as cookies, user preference settings, and cached resources. All the WebView2 controls associated with the same profile share the single profile folder. 
+
+Note that the profile maps to the same concept of Edge browser profile.
+
+WebView2 multiple profile API allows you to create and manipulate profiles to work with your WebView2 controls to fulfill your needs.
+
+
+<!-- ====================================================================== -->
+## Specify the profile when creating WebView2
+
+This method creates an options object to give specific information of a profile, including `ProfileName` and `IsInPrivateModeEnabled`. This object is to be used to specify the target profile when creating a WebView2 control.
```

Comment:
Note that "This method" is confusing, this first section is about:
CreateCoreWebView2ControllerOptions
and the next section is about an almost identical method name and also says "This method":
CreateCoreWebView2ControllerWithOptions

In my forked PR into this PR, I'm trying adding h4 headings to differentiate while introducing these two confusingly similarly named methods.

Reply:
Done, see
https://github.com/MicrosoftDocs/edge-developer/pull/2245/files

Please close this comment thread.